_pages/docs/reports/may2022.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: PyHC project status, May 2022
+permalink: /docs/reports/may2022/
+exclude: true
+---
+
+**Meeting/Workshops**
+
+* We held our eighth-overall and fifth all-virtual PyHC Spring 2022 meeting. 
+    * The meeting took place Monday, May 16th - Thursday, May 19th (on Zoom, from 9 - 11 AM MT). 
+    * Presentations included PyHC projects updates (in particular, core projects), project introductions from the HERMES SOC and pysat's pysatMissions, as well as presentations on other PyHC community efforts (the Space Collection and an executable paper that leverages PyHC packages). Additionally, meeting attendees participated in hackathons for several projects, where the goal was to create a Gallery example on integrating a package with at least one other PyHC package. Finally, other relevant community discussions also took place on the final day (future funding opportunities, best practices in the community and future project considerations). 
+    * Meeting report to come after the summer school finishes.
+* We wrapped up planning for the [Inaugural PyHC 2022 Summer School](https://heliopython.org/summer-school), which kicked off as of today!
```

Comment:
One thing:

- We have "which kicked off as of today!" on line 14,
- but then "the event is taking place in... from..." on line 16.

Two conflicting time tenses. What date is this supposed to be submitted? I'd either remove "which kicked off as of today" or change the other spot to "the event took place in... from..."

Reply:
Ignoring :D